src/workflows/airqo_etl_utils/data_validator.py

```
@@ -98,8 +105,66 @@ def format_data_types(
+        """
+
+        if value is None:
+            return None
+
+        if isinstance(value, str):
+            try:
+                parsed = ast.literal_eval(value)
+                if isinstance(parsed, list):
+                    if all(isinstance(item, dict) for item in parsed):
+                        return parsed
+            except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)